public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -57,7 +65,73 @@ export function transform(
   };
   const prometheusResult = response.data.data;
 
-  if (!prometheusResult.result) {
+  if (isExemplarData(prometheusResult)) {
+    const events: ExemplarsDataFrameViewDTO[] = [];
+    prometheusResult.forEach(exemplarData => {
+      const data = exemplarData.exemplars.map(exemplar => {
+        return {
+          time: exemplar.scrapeTimestamp,
+          component: getExemplarInfoComponent(
```

Comment:
@dprokop @zoltanbedi Was looking at a way to allow internal links for this and wanted to ask if passing a component will be allowed for all plugins? If so we will have to have some interface that this should conform to and pass some additional props to it. Mainly a function that will turn a dataLink into clickable link with proper href and onClick handler in case it's an internal link.